pkg/cd4pe/.openapi-generator/VERSION

```diff
@@ -1 +0,0 @@
-4.2.0
```

Comment:
It might be useful to keep this file so others can see which version of openapi-generator was used.

Or else add the version in README.md, for example:
> This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project v4.2.0

Reply:
there is a .openapi-generator/VERSION file at the root of the project that has the version - i think this file had been originally moved to the top level but cant be sure as there was only ever one commit but there was no configuration/options in the generator to specify any other paths that the pkg/cd4pe directory so i am assuming it had been

Reply:
Ok, I see that now - thanks